### DISCUSSION TITLE
Hyperparameters tuning

### DISCUSSION THREAD

**Discussion by alibek-wand** [autogluon/autogluon#2252] [Q&A] [answered]
Hi!

Are hyperparameters of models are being tuned by default in the training process?
For example, if I write:
TabularPredictor(label=label, problem_type="regression").fit(train_data=train_data,
                    hyperparameters={"GBM": {}, "LR": {}})
will it tune some hyperparameters of LightGBM and Linear model?

In the documentation I found "hyperparameters_tune = True" parameter, but I could not figure out where to use it 
https://auto.gluon.ai/dev/_modules/autogluon/tabular/predictor/predictor.html#TabularPredictor.fit
Is this parameter True by default?

Thank you!

**Comment by Innixma** [comment, marked as answer]
It is not enabled by default, you'd have to use `hyperparameter_tune_kwargs='auto'` to enable tuning. Note that some models will error in the latest release during HPO (Random Forest), so you'd want to use the latest pre-release when doing HPO.

**Reply by Innixma** [reply]
I opened #2253 to fix the incorrect reference to `hyperparameter_tune`, thanks for notifying!
